model/src/main/java/org/mskcc/cmo/metadb/model/web/PublishedMetadbSample.java

```diff
@@ -22,6 +22,7 @@
     @Convert(UuidStringConverter.class)
     private UUID metaDbSampleId;
     private UUID metaDbPatientId;
+    private String igoRequestId;
```

Comment:
~~note for self: still unsure about needing to add these props to the sample schema.. will think about this further.~~

edit: see proposed requested change below

Reply:
[requested change]  `SampelMetadata` already contains `additonalProperties`, a hashmap of string/string. We can add _that_ to `PublishedMetadbSample` and just add this information (igoRequestId and isCmoSample) to the `additionalProperties` map.

Reply:
the request id is at the request-level, not sample level